Userland/Libraries/LibGfx/ImageFormats/CCITTDecoder.cpp

```diff
@@ -316,6 +316,16 @@ ErrorOr<void> decode_single_ccitt3_1d_line(BigEndianInputBitStream& input_bit_st
     return {};
 }
 
+static ErrorOr<void> read_eol(BigEndianInputBitStream& bit_stream)
+{
+    constexpr u16 EOL = 0x000000000001;
```

Comment:
nit: `0x0000'0000'0001` is easier to read. Also, an u16 can only store 16 bits, so maybe drop the leading four zeros? Or maybe you meant `0b` instead of `0x`?

Reply:
So many wrong things in so few characters 🫣